build-conventions-secant/src/main/kotlin/.editorconfig

```diff
@@ -1,11 +0,0 @@
-disabled_rules=import-ordering,no-wildcard-imports
```

Comment:
Why did we removed this whole file?

Reply:
I don’t think this file was being used, as the ktlint convention plugin is loading a different path.